packages/xo-server/src/_extractIpFromVmNetworks.js

```diff
@@ -1,8 +1,8 @@
 // See: https://github.com/xapi-project/xen-api/blob/324bc6ee6664dd915c0bbe57185f1d6243d9ed7e/ocaml/xapi/xapi_guest_agent.ml#L59-L81
 //
-// Returns <min(n)>/ipv4/<min(m)> || <min(n)>/ipv6/<min(m)> || undefined
+// Returns <min(n)>/ip || <min(n)>/ipv4/<min(m)> || <min(n)>/ipv6/<min(m)> || undefined
 // where n corresponds to the network interface and m to its IP
-const IPV4_KEY_RE = /^\d+\/ipv4\/\d+$/
+const IPV4_KEY_RE = /^\d+\/ip(v4\/\d+)?$/
```

Comment:
In RE, parentheses are used for two things: grouping and capturing.

When you don't need capturing, you can use a *non-capturing group*, which is denoted by the prefix `?:`:

```suggestion
const IPV4_KEY_RE = /^\d+\/ip(?:v4\/\d+)?$/
```